azure-pipelines.yml

```diff
@@ -13,12 +13,52 @@ resources:
     name: 1ESPipelineTemplates/1ESPipelineTemplates
     ref: refs/tags/release
 
+variables:
+- name: system.debug
```

Comment:
Assuming we don't want to keep this as true and was only meant to be used for debugging?

Reply:
Retained from existing code, but I can drop this.